### PR TITLE
Ping the location channel when someone shouts out successfully

### PR DESCRIPTION
**Pull request by chriscarter90**

### CODE REVIEW

**Comment by FriedSock** [comment]
tests?

File: app/controllers/shoutouts_controller.rb [hunk 1]
```diff
@@ -60,6 +60,28 @@ def create
...
 
+    if shoutout.tag_list.include?("london")
+      slack_client.chat_postMessage(
+        channel: Settings.slack.london,
+        text: "@#{params[:user_name]} cheered: #{message}\nSee it, and other cheers at #{root_url}!",
+        as_user: true
+      )
+    end
+
+    if shoutout.tag_list.include?("capetown")
```
Comment (otlaitil) [inline app/controllers/shoutouts_controller.rb]: @chriscarter90 can the tag list include both London and CT at once? Just wondering, I am not familiar with the app nor the codebase.

Comment (chriscarter90) [inline app/controllers/shoutouts_controller.rb]: Yeah it can.

**Comment by chriscarter90** [comment]
@FriedSock We don't need tests where we're goin'.

**Comment by otlaitil** [comment]
:+1:

**Comment by FriedSock** [comment]
:+1:

**Pull request merged**